OpenQA/Isotovideo/Runner.pm

```diff
@@ -19,16 +19,29 @@ use autotest ();
 use needle ();
 use commands ();
 use distribution ();
+use Mojo::File qw(path);
+
+# Avoid "Subroutine JSON::PP::Boolean::(0+ redefined" warnings
+# Details: https://progress.opensuse.org/issues/90371
+use JSON::PP;
```

Comment:
That shouldn't be moved here. It's something that should happen before all other module use statements.

Reply:
If it is actually need in this module, then copy that use statement, but leave the original in the isotovideo script

Reply:
@b10n1k Did you see this? To me it looks like you didn't change it.

I recommend always checking and responding to unresolved conversations.

Reply:
Are we good here then?

Reply:
for me it doesnt work if i remove it. So i did copy it and put it back in isotovideo as suggested.